ITKIOScanco_UnitConversion.ipynb

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.    del all_AIM_filenames, all_ISQ_filenames
</pre><p>We can remove the <code>del</code> -- this is a Matlab-ism due to poor scoping in Matlab's lanugage. We generally do not need it Python. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/KitwareMedical/ITKIOScanco/pull/74/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>